packages/@coorpacademy-components/src/organism/list-item/index.js

```diff
@@ -1,33 +1,58 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import {isEmpty, map} from 'lodash/fp';
+import {isEmpty, map, noop, get} from 'lodash/fp';
+import {convert} from 'css-color-function';
 import classnames from 'classnames';
+import Provider from '../../atom/provider';
 import Tag from '../../atom/tag';
+import Icon from '../../atom/icon';
 import ButtonLink from '../../atom/button-link';
 import BulletPointMenuButton from '../../molecule/bullet-point-menu-button';
 // eslint-disable-next-line css-modules/no-unused-class
 import style from './style.css';
 
-const ListItem = ({
-  buttonLink,
-  secondButtonLink,
-  bulletPointMenuButton,
-  tags,
-  title,
-  dataColumns,
-  order,
-  'aria-label': ariaLabel,
-  contentType,
-  isBulkStyle = false,
-  isOverflowHidden = false
-}) => {
+const ListItem = (
+  {
+    buttonLink,
+    secondButtonLink,
+    bulletPointMenuButton,
+    tags,
+    title,
+    selected,
+    subtitle,
+    dataColumns,
+    order,
+    'aria-label': ariaLabel,
+    disabled = false,
+    contentType,
+    isBulkStyle = false,
+    isOverflowHidden = false,
+    onClick = noop
+  },
+  context
+) => {
+  const {skin} = context;
+  const primarySkinColor = get('common.primary', skin);
```

Comment:
we should double check this has no impact on component already using this one

Reply:
Hi Adrien, this only associated with the newly introduced `selected` props, so it wont affect any older use cases

Reply:
indeed 😅